eiffel-vocabulary/EiffelAnnouncementAcknowledgedEvent.md

```diff
@@ -19,7 +19,36 @@
 The EiffelServiceDeployedEvent is to declare that an EiffelAnnouncementPublishedEvent has been received & acknowledged by a person or a machine.
```

Comment:
Copy-paste mishap? AnnouncementAcknowledge vs. ServiceDeployed

(This must has been slipped through earlier)